examples/functions/golang/grpc-serving/doc.go

```diff
@@ -0,0 +1,16 @@
+/*
+FrontendAPI contains the unique files required to run the API endpoints for
```

Comment:
This comment is all wrong.

Reply:
I thought i deleted this file (currently examples currently dont have doc files). I have now deleted it.